scripts/build/config.js

```diff
@@ -66,6 +67,14 @@ const parsers = [
   {
     input: "src/language-html/parser-parse5.js",
     target: "node"
+  },
+  {
+    input: "src/language-yaml/parser-yaml.js",
+    target: "universal",
+    bundler: "webpack",
+    plugins: [
+      require.resolve("./babel-plugins/replace-array-includes-with-indexof")
```

Comment:
1) I thought babel was including polyfills, looks like it isn't. Maybe we should set it up?

2) This should probably be named `babelPlugins` to be more specific about what kind of plugin this is (we could also have rollup or webpack plugins)